tests/helpers/gen_write_load.tcl

```diff
@@ -2,17 +2,21 @@ source tests/support/redis.tcl
 
 set ::tlsdir "tests/tls"
 
-proc gen_write_load {host port seconds tls} {
+proc gen_write_load {host port seconds tls key} {
```

Comment:
1. consider using empty string instead of "0"
2. I think it would be better to have default value for key (which is the empty string)

Reply:
1. I will fix it.
2. `gen_write_load` is already used in many places without key name parameter, generating random keys. I don't want to remove this functionality.